pkgs/development/libraries/qt-5/5.9/qtnetworkauth.nix

```diff
@@ -0,0 +1,10 @@
+{ qtSubmodule, qtbase }:
+
+qtSubmodule {
+  name = "qtnetworkauth";
+  qtInputs = [ qtbase ];
+  outputs = [ "out" "dev" "bin" ];
```

Comment:
@ilpianista As is, the build does not produce any QML outputs. If that is correct, please remove the `bin` output and the `postInstall` hook below. If the build *should* produce QML outputs, add `qtdeclarative` to the `qtInputs`.

Reply:
@ttuegel It doesn't produce any QML output. Fixed, thanks!